AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -814,6 +814,15 @@ protected Void doInBackground(Void... params) {
               Constants.loggingEnabled = true;
             }
 
+            JSONArray enabledCounters = response.optJSONArray(
+                    Constants.Keys.ENABLED_COUNTERS);
+            if (enabledCounters != null) {
+              for (int i=0; i < enabledCounters.length(); i++) {
+                String name = enabledCounters.getString(i);
+                Log.enableCounter(name);
+              }
+            }
+
```

Comment:
This `handleStartResponse` is already wayyyy too long. Can you take this bit of code out to something like `checkCounters(response)` or something.